sections/results/results.tex

```diff
@@ -70,43 +67,29 @@ \section{Results}\label{sec: results}
-
  \begin{figure}[H]
   \begin{center}
     \includegraphics[width=\linewidth]{../../images/combinedCircuit.png}
   \end{center}
-  \caption{The combined circuit. Note that the objective qubit here is qubit 3. Evaluation qubits are chosen for pragmatical reasons. For results a register consisting of seven qubits have been used.}
+  \caption{The combined circuit. Note that the objective qubit here is qubit 3. 
+  Three evaluation qubits are chosen for pragmatical reasons. For results a register consisting of seven qubits have been used.}
   \label{fig:combinedCircuit}
 \end{figure}
-The combined Circuit is displayed in figure $\ref{fig:combinedCircuit}$. For the obtained results, i.e. the estimate of Amplitude Estimation seven qubits in the evaluation register have been used. Measuring all qubits in the evaluation register gives the following result:
+Using the grover operator the combined circuit is displayed in figure $\ref{fig:combinedCircuit}$. 
+For the obtained results, i.e. the estimate $\tilde{a}$ seven qubits in the evaluation register have been used. 
+Measuring all qubits in the evaluation register gives the following result:
 \begin{align}
     &\text{Exact Value: } &0.1133\\
     &\text{Estimated Value: } &0.1061\\
     &\text{95 Percent Confidence interval:	} &[0.1157, 0.1209]
 \end{align}
-Therefore the estimated value is close to the exact value. One problem here is that the estimated value does not lie within the confidence interval.
-Note that the estimated value is the mean of applying eq. $\ref{eq:estimate_a}$ to the results of amplitude estimation. Additionally a post processing has been added in order to map from $[0,1]$ to the domain of stock prices. It is possible to apply a maximum likelihood estimator to the estimated value: 
-\begin{equation}
-    \text{MLE estimator value: } 0.1189.
-\end{equation}
-
-The value of the MLE estimator vaue is slightly better than the value of the ordinary estimator. Furthermore it lies within the confidence interval which is not the case for the estimated value.  
-
- \begin{figure}[H]
-  \begin{center}
-    \includegraphics[width=\linewidth]{../../images/resultAE.png}
-  \end{center}
-  \caption{Results of Canonical AE. Red dotted line show the exact value. Green line is the MLE estimator value. }
-  \label{fig:resultAE}
-\end{figure}
-When we now execute the same Circuit using Iterative Amplitude Estimation \cite{Grinko_2021} from qiskit we get the following results:
+Therefore the estimated value is close to the exact value.//
+When we now execute the same circuit for the payoff using iterative amplitude estimation \cite{Grinko_2021}  (i.e. without phase estimation) from qiskit we get the following results:
 \begin{align}
     &\text{Exact Value: } &0.1133\\
-    &\text{Estimated Value: } &0.1203\\
-    &\text{95 Percent Confidence interval:	} &[0.1153, 0.1254]
+    &\text{Estimated Value: } &0.1676\\
+    &\text{95 Percent Confidence interval:	} &[0.1615, 0.1737]
```

Comment:
Is the result from iterative AE indeed this much different (far outside confidence interval)? 
In the text you say it is comparable with the previous result, but I am not sure that is the case. This, we can perhaps discuss during the next meeting if needed.

Reply:
Of course we can discuss